src/pybind/mgr/rbd_support/schedule.py

```diff
@@ -387,17 +387,18 @@ def load(self,
 
         schedule_cfg = self.handler.module.get_module_option(
             self.handler.MODULE_OPTION_NAME, '')
+        schedule_local_cfg = self.handler.module.get_localized_module_option(
+             self.handler.MODULE_OPTION_NAME, '')
```

Comment:
I would move this below the comment below, as it is part of that and we will remove that whole block with the comment  in some future.

Reply:
Done